lib/rails_api_auth/engine.rb

```diff
@@ -1,3 +1,4 @@
+require 'httparty'
```

Comment:
This means that `httparty` will now always be required regardless of whether the `Facebook` or `Google` services are actually used or not. Also this change is unrelated to the SSL stuff.